README.md

```diff
@@ -121,6 +122,7 @@ the only label value published for features is the string `"true"`._
   "feature.node.kubernetes.io/selinux-<feature-name>": "true",
   "feature.node.kubernetes.io/storage-<feature-name>": "true",
   "feature.node.kubernetes.io/<hook name>-<feature name>": "<feature value>"
```

Comment:
Should there a comma after  feature.node.kubernetes.io/< hook name >-< feature name >: < feature value > before the system-< feature-name> line?

Reply:
Correct, well spotted! I had pasted the new line in wrong, as the "hook" line should be the last on the list. Fixed now